### DISCUSSION TITLE
Google translate: How to filter multilingual data in a database?

### DISCUSSION THREAD

**Discussion by Rodien** [2sic/2sxc#3035] [Q&A] [answered]
Hi guys,

I am writing a query to retrieve the 'News' data, which is multilingual. Currently, I am retrieving data but I am not getting the default language, so I would like to filter it to get data in a different language.

I assume there must be a language key somewhere in the database tables, but I cannot find it.

Therefore, at the moment, I am returning the following data, but I want to retrieve it in a specific language.

![image](https://user-images.githubusercontent.com/50739829/224682844-11c2bcd1-554d-4be3-b4c1-a9997c671f59.png)

I want to use the query inside DNN Sharp searchboost, which is the reason why I am doing this.

@tvatavuk @iJungleboy

**Comment by iJungleboy** [comment, marked as answer]
Hi @Rodien 

The SQL structure is fairly difficult to work with. Especially with multi-language. 

If there is any way to provide data to SearchBoost using a C# API I would recommend that. For example, IMHO SearchBoost support the standard Dnn `ISearchable` interface which 2sxc supports as well - you can even customize what it does. I would highly recommend you use that instead. 

The Sql method is challenging. Basically you have

1. Entities
2. Attributes
3. Values - connected to Dimensions - which are an ID referencing a language...

**Reply by Rodien** [reply]
Oh okay, I am going to take a look and try a few things.
